src/pages/collections/_type/_.vue

```diff
@@ -78,27 +94,29 @@
   import { BASE_URL as EUROPEANA_DATA_URL } from '../../../plugins/europeana/data';
   import { getEntityTypeHumanReadable, getEntitySlug, getEntityUri } from '../../../plugins/europeana/entity';
   import { langMapValueForLocale } from  '../../../plugins/europeana/utils';
+  import EntityUpdateModal from '../../../components/entity/EntityUpdateModal';
```

Comment:
As this will not be visible to most users, lazy-load it, as with BrowseSections and RelatedCollections